abi/sector.go

```diff
@@ -284,6 +298,41 @@ var SealProofInfos = map[RegisteredSealProof]*SealProofInfo{
 		WindowPoStProof:  RegisteredPoStProof_StackedDrgWindow64GiBV1,
 		UpdateProof:      RegisteredUpdateProof_StackedDrg64GiBV1,
 	},
+	RegisteredSealProof_StackedDrg2KiBV1_1_Feat_NiPoRep: {
+		ProofSize:        192,
```

Comment:
we'd better get these right before merging this

Reply:
Updated the proofs-sizes here: https://github.com/filecoin-project/go-state-types/pull/269/commits/2763495cb7447abc95bb54059ff3cd11f07cbfde